Bottom: a0d7f878f73dfd0d4d6bc1521cf534a9356f9247
Top:    02ba21569834d698cca7eb5533e92989da5b373e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:14:21 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 717b121..71c8a08 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -243,7 +243,6 @@ get_contour_coord_n_in_mm (PLINE *contour, int n, double *x, double *y)
   *y = COORD_TO_MM (vertex->point[1]); /* FIXME: PCB's coordinate system has y increasing downwards */
 }
 
-
 object3d *
 object3d_from_board_outline (void)
 {
